cauldron/sql.py

```diff
@@ -125,10 +130,11 @@ class PostgresStore:
     _WHERE_AND = '{} {} %s'
     _PLACEHOLDER = ' %s,'
     _COMMA = ', '
+    _pool_pending = asyncio.Semaphore(1)
 
     @classmethod
     def connect(cls, database: str, user: str, password: str, host: str, port: int, *, use_pool: bool=True,
-                enable_ssl: bool=False, minsize=1, maxsize=50, keepalives_idle=5, keepalives_interval=4, echo=False,
+                enable_ssl: bool=False, minsize=1, maxsize=30, keepalives_idle=5, keepalives_interval=4, echo=False,
```

Comment:
Not sure why this change is essential. The consumer can override the default behaviour.